Doc/glossary.rst

```diff
@@ -159,14 +159,14 @@ Glossary
       :class:`str` objects.
 
    borrowed reference
-      In the Python's C API, a borrowed reference is a reference to an object.
+      In Python's C API, a borrowed reference is a reference to an object.
       It does not modify the object reference count. It becomes a dangling
       pointer if the object is destroyed. For example, a garbage collection can
       remove the last :term:`strong reference` to the object and so destroy it.
 
       Calling :c:func:`Py_INCREF` on the :term:`borrowed reference` is
       recommended to convert it to a :term:`strong reference` in-place, except
-      if the object cannot be destroyed before the last usage of the borrowed
+      when the object cannot be destroyed before the last usage of the borrowed
```

Comment:
Please update also Doc/c-api/weakref.rst which uses the same sentene.

Reply:
Oops, thanks for noticing that and telling me.